extension/scripts/features/page-jump/ttPageJump.js

```diff
@@ -0,0 +1,698 @@
+"use strict";
+
+(async () => {
+	const devices = await checkDevice();
+	if (devices.mobile || devices.tablet) return "Not supported on mobiles or tablets!";
+
+	featureManager.registerFeature(
+		"Page Jump",
+		"global",
+		() => settings.pages.global.pageJump,
+		undefined,
+		initializeJumper,
+		deinitializeJumper,
+		{
+			storage: ["settings.pages.global.pageJump", "settings.scripts.pageJump.maxPages"],
+		},
+		undefined
+	);
+
+	const userId = document.find('.settings-menu a[href*="profiles"]').href.split("=")[1];
```

Comment:
This errors if there is no header (eg [here](https://www.torn.com/loader.php?sid=holdemFull)). 
Was this an attempt to work around the API requirement of `userdata.player_id`?
If so, best solution here may be something like:
```js
const userId = userdata?.player_id ?? document.find(...)?.href.split("=")[1];
if (!userId) return "User ID could not be found";
```
Or alternatively just requiring API access in the `requirements` arg of `featureManager.registerFeature` or only dropping the "My Profile" row when the user ID cannot be found?

Reply:
I honestly wasn't sure of the best way to get the playerID. I searched the codebase for some examples but it seemed to be pretty mixed in how it is done. I ultimately need it to load your own profile, so yeah I do like your suggestion and I'll update that later when I am back at the computer.

Reply:
Alright, updated. I didn't want to make the whole feature dependant on API access, so I went the route of first trying via the API, then via the selector, and worst case scenario the "My Profile" row/entry is hidden from the results.